dipy/stats/analysis.py

```diff
@@ -157,14 +128,40 @@ def dti_measures(bundle, metric, dt, pname, bname, subject, group, ind, dir):
                              order=1)
 
     dt["disk#"].extend(ind[list(range(len(values)))]+1)
-    dt["bundle"].extend([bname]*len(values))
+    #dt["bundle"].extend([bname]*len(values))
     dt["subject"].extend([subject]*len(values))
-    dt["group"].extend([group]*len(values))
+    dt["group"].extend([group_id]*len(values))
     dt[pname].extend(values)
+    
+    
+    for st_i in range(len(bundle)):
+    
+        st = bundle[st_i]
+        dt["streamline"].extend([st_i]*len(st))
+        
+    
+
+    #devide and conquer
+    file_name = bname+"_"+pname
 
-    _save_hdf5(os.path.join(dir, pname), dt, col_name="bundle")
 
+    _save_hdf5(os.path.join(dir, file_name), dt, col_name="bundle")
 
+def assignment_map(target_bundle, model_bundle, no_disks):
+
+    mbundle_streamlines = set_number_of_points(model_bundle,
+                                           nb_points=no_disks)
+
+    metric = AveragePointwiseEuclideanMetric()
+    qb = QuickBundles(threshold=85., metric=metric)
```

Comment:
I deliberately made this static with such a high threshold value of 85 because right now we are only supporting assignments created with one cluster. If we make it a parameter and someone sets the threshold too low, it will generate many clusters per model bundle.

It would be good to support more than one cluster centroids for creating assignment maps. Maybe we can add this functionality later when we have some statistical method that uses more than one cluster centroids for creating bundle segments.